easybuild/easyconfigs/g/gputools/gputools-0.28-e5cb024-ictce-5.3.0-R-3.0.2.eb

```diff
@@ -1,30 +1,27 @@
-name = 'Rgputools'
-version = '0.28'
-versionsuffix = '-e5cb024'
-
 easyblock = 'RPackage'
 
+name = 'gputools'
+version = '0.28'
+commit = 'e5cb024'
+versionsuffix = '-%s-R-%%(rver)s' % commit
+
 homepage = 'https://github.com/nullsatz/gputools/wiki'
 description = """This package provides R interfaces to a handful of common functions implemented
  using the Nvidia CUDA toolkit. Some of the functions require at least GPU Compute Capability 1.3."""
 
 toolchain = {'name': 'ictce', 'version': '5.3.0'}
 
-sources = ['%s.tar.gz' % versionsuffix[1:]]
 source_urls = ['https://github.com/nullsatz/gputools/archive']
+sources = ['%s.tar.gz' % commit]
```

Comment:
let's add a hash here that corresponds to the stated git commit; I speak from experience and big trouble
(merging anyway)